packages/kbn-securitysolution-utils/src/path_validations/index.test.ts

```diff
@@ -65,32 +84,55 @@ describe('validateFilePathInput', () => {
         : OperatingSystem.LINUX;
 
     it('does not warn on valid filenames', () => {
-      expect(validateFilePathInput({ os, value: '/opt/*/FILENAME.EXE-1231205124.gz' })).not.toEqual(
-        FILENAME_WILDCARD_WARNING
-      );
       expect(
-        validateFilePathInput({
+        validateWildcardInput({
+          field: 'file.path.text',
+          os,
+          value: '/opt/*/FILENAME.EXE-1231205124.gz',
+        })
+      ).not.toEqual(WILDCARD_WARNING);
+      expect(
+        validateWildcardInput({
+          field: 'file.path.text',
           os,
           value: "/opt/*/test$  as2@13---12!@#A,DS.#$^&$!#~ 'as'd.华语.txt",
         })
-      ).not.toEqual(FILENAME_WILDCARD_WARNING);
+      ).not.toEqual(WILDCARD_WARNING);
     });
     it('warns on wildcard in file name at the end of the path', () => {
-      expect(validateFilePathInput({ os, value: '/opt/bin*' })).toEqual(FILENAME_WILDCARD_WARNING);
-      expect(validateFilePathInput({ os, value: '/opt/FILENAME.EXE-*.gz' })).toEqual(
-        FILENAME_WILDCARD_WARNING
+      expect(validateWildcardInput({ field: 'file.path.text', os, value: '/opt/bin*' })).toEqual(
+        WILDCARD_WARNING
+      );
+      expect(
+        validateWildcardInput({ field: 'file.path.text', os, value: '/opt/FILENAME.EXE-*.gz' })
+      ).toEqual(WILDCARD_WARNING);
+    });
+
+    it('warns on wildcard on fields that are not file paths', () => {
+      expect(validateWildcardInput({ field: 'event.category', os, value: '*' })).toEqual(
+        WILDCARD_WARNING
       );
     });
 
     it('warns on windows paths', () => {
-      expect(validateFilePathInput({ os, value: 'd:\\path\\file.exe' })).toEqual(FILEPATH_WARNING);
+      expect(
+        validateWildcardInput({ field: 'file.path.text', os, value: 'd:\\path\\file.exe' })
+      ).toEqual(FILEPATH_WARNING);
     });
 
     it('warns on malformed paths', () => {
-      expect(validateFilePathInput({ os, value: 'opt/bin\\file.exe' })).toEqual(FILEPATH_WARNING);
-      expect(validateFilePathInput({ os, value: '1242' })).toEqual(FILEPATH_WARNING);
-      expect(validateFilePathInput({ os, value: 'w12efdfa' })).toEqual(FILEPATH_WARNING);
-      expect(validateFilePathInput({ os, value: '/folder/' })).toEqual(FILEPATH_WARNING);
+      expect(
+        validateWildcardInput({ field: 'file.path.text', os, value: 'opt/bin\\file.exe' })
+      ).toEqual(FILEPATH_WARNING);
+      expect(validateWildcardInput({ field: 'file.path.text', os, value: '1242' })).toEqual(
+        FILEPATH_WARNING
+      );
+      expect(validateWildcardInput({ field: 'file.path.text', os, value: 'w12efdfa' })).toEqual(
+        FILEPATH_WARNING
+      );
+      expect(validateWildcardInput({ field: 'file.path.text', os, value: '/folder/' })).toEqual(
+        FILEPATH_WARNING
```

Comment:
I would suggest that we create new tests for `validateWildcardInput` instead of changing these. Those would test for two cases 1. when field is `file.path.text` and 2. for any other field not `file.path.text`.